### PR TITLE
3 - naked templates

### PR DESCRIPTION
**Pull request by WesJourdan**
- Remove styles and remove references to global components that no longer exist.

NOTE: Intentionally left styling on Single.Skeleton since it matches the base layout.

### CODE REVIEW

File: lib/v_3.0/templates/web/views/Single.jsx [hunk 1]
```diff
@@ -33,25 +32,22 @@ const Single__PascalName__ = () => {
...
+      <Link to={`${location.pathname}/update`}>Update __PascalName__</Link>
     </__PascalName__Layout>
   )
 }
 
 const Skeleton = () => {
   return (
     <div className="animate-pulse">
-      <div className="w-48 h-5 bg-gray-400"/>
+      <p className="w-48 h-5 bg-gray-400"/>
```
Comment (grantfowler) [inline lib/v_3.0/templates/web/views/Single.jsx:47]: tsk tsk

**Review by grantfowler — APPROVED** [review]

File: lib/v_3.0/templates/web/components/Form.jsx [hunk 2]
```diff
@@ -52,19 +51,19 @@ const __PascalName__Form = ({
...
           <div className="">
-            <Button
+            <button
               disabled={disabled}
               link={cancelLink}
-              skin='white'
+              type='button'
             >
               Cancel
-            </Button>
```
Comment (ffugly) [inline lib/v_3.0/templates/web/components/Form.jsx:61]: this should be a `<Link to={cancelLink}>Cancel</Link>`

Comment (ffugly) [reply lib/v_3.0/templates/web/components/Form.jsx:61]: also remove the two wrapping `<div className="">`

File: lib/v_3.0/templates/web/views/List.jsx [hunk 3]
```diff
@@ -32,24 +30,19 @@ const __PascalName__List = () => {
...
     <__PascalName__Layout title={'__PascalName__ List'}>
-      <div className="flex w-full mb-4 justify-end">
-        <Button
-          link='/__kebabNamePlural__/new'
-          size="sm"
-          skin="secondary"
+       <h1>__PascalName__ List</h1>
+        <div className="">
+          <Link to="/__camelNamePlural__/new">New __PascalName__</Link>
+        </div>
```
Comment (ffugly) [inline lib/v_3.0/templates/web/views/List.jsx:36]: remove wrapping `<div className="">`

**Pull request merged**